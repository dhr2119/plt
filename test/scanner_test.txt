+ - * / { } [ ] ( ) hi 99 int string null void
