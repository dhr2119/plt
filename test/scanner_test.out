PLUS
MINUS
TIMES
DIVIDE
LBRACE
RBRACE
LBRACKET
RBRACKET
LPAREN
RPAREN
ID
LITERAL
INT
ID
NULL
VOID
